polaris/_mixins.py

```diff
@@ -68,6 +64,9 @@ def verify_checksum(self, md5sum: str | None = None):
         self.md5sum = self._compute_checksum()
 
         if self.md5sum != md5sum:
+            # Import here to prevent circular imports
+            from polaris.utils.errors import PolarisChecksumError
```

Comment:
I think we should still create the `mixins` folder unless we have a strong preference not to. Inline imports can get messy if the codebase grows and we become ok with doing this elsewhere.